cli.md

```diff
@@ -2,18 +2,18 @@
 
 Supported commands:
 
-| **Command**                 | **Subcommand**                                          | **Description**                                               |
-|-----------------------------|---------------------------------------------------------|---------------------------------------------------------------|
-| [entrypoints](#entrypoints) |                                                         | Determine the entrypoint directories to run Guardian commands |
-| apply                       | [run](#apply-run)                                       | Run Terraform apply for a directory                           |
-| plan                        | [run](#plan-run)                                        | Run Terraform plan for a directory                            |
-| [run](#run)                 |                                                         | Run a Terraform command for a directory                       |
-| iam                         | [cleanup](#iam-cleanup)                                 | Remove any expired IAM in a GCP organization                  |
-|                             | [detect-drift](#iam-detect-drift)                       | Detect IAM drift in a GCP organization                        |
-| drift                       | [statefiles](#drift-statefiles)                         | Detect drift for terraform statefiles                         |
-| workflows                   | [plan-status-comment](#workflows-plan-status-comment)   | Add Guardian plan comment to a pull request                   |
-|                             | [remove-plan-comments](#workflows-remove-plan-comments) | Remove previous Guardian plan comments from a pull request    |
-|                             | [validate-permissions](#workflows-validate-permissions) | Validate required permissions for the current GitHub workflow |
+| **Command**                 | **Subcommand**                                          | **Required Github Permission**                              | **Description**                                               |
+|-----------------------------|---------------------------------------------------------|-------------------------------------------------------------|---------------------------------------------------------------|
+| [entrypoints](#entrypoints) |                                                         | `contents: read`, `pull-requests: write`                    | Determine the entrypoint directories to run Guardian commands |
```

Comment:
nit: I really like this approach, but this would read much better if we could add a newline between each permission needed.....not sure if we can do that with an HTML <br/> or what is supported

Reply:
Just realized the `<br />` was rendered

Reply:
Adding a `<br>` tag appears to do the trick.